src/client/dfuse/ops/lookup.c

```diff
@@ -197,18 +198,18 @@ dfuse_cb_lookup(fuse_req_t req, struct dfuse_inode_entry *parent,
 		const char *name)
 {
 	struct dfuse_projection_info	*fs_handle = fuse_req_userdata(req);
-	struct dfuse_inode_entry	*ie = NULL;
+	struct dfuse_inode_entry	*ie;
 	int				rc;
 	char				out[DUNS_MAX_XATTR_LEN];
 	char				*outp = &out[0];
 	daos_size_t			attr_len = DUNS_MAX_XATTR_LEN;
 
-	DFUSE_TRA_DEBUG(fs_handle,
+	DFUSE_TRA_DEBUG(parent,
 			"Parent:%#lx '%s'", parent->ie_stat.st_ino, name);
 
 	D_ALLOC_PTR(ie);
 	if (!ie)
-		D_GOTO(out, rc = ENOMEM);
+		D_GOTO(out_free, rc = ENOMEM);
```

Comment:
why out_free if alloc failed?

Reply:
This looks like a bad merge actually, I'll change this bit back.